Makefile

```
@@ -92,4 +92,9 @@ help: ## Display this help screen.
 test-scanner:
 	make test -C $(SCANNERS_DIR)/$(target)
 test-hook:
-	make test -C $(HOOKS_DIR)/$(target)		
\ No newline at end of file
+	make test -C $(HOOKS_DIR)/$(target)		
+
+
+.PHONY:
+megalint:
```

```suggestion
lint:
```
Let's try to keep our makefile targets independent from the technology implementation